nemo/collections/tts/g2p/models/t5.py

```diff
@@ -271,3 +272,48 @@ def setup_multiple_test_data(self, test_data_config: Union[DictConfig, Dict] = N
     @classmethod
     def list_available_models(cls) -> 'List[PretrainedModelInfo]':
         return []
+
+    def _prepare_for_export(self, **kwargs):
+        super()._prepare_for_export(**kwargs)
+
+        tensor_shape = ('B', 'T')
+
+        # Define input_types and output_types as required by export()
+        self._input_types = {
+            "input_ids": NeuralType(tensor_shape, TokenIndex()),
+        }
+        self._output_types = {
+            "preds_str": NeuralType(tensor_shape, LabelsType()),
```

Comment:
If I am not mistaken the term `preds_str` in the `ctc.py` class is referring to a string, whereas here its returning an index. `pred_tokens` or `pred_indices` might be a better name.